ui/src/app/modules/status/widgets/update-info-widget/update-info-widget.component.ts

```diff
@@ -178,4 +197,39 @@ export class UpdateInfoWidgetComponent implements OnInit {
       this.$toastr.error(error.message, this.$translate.instant('toast.title_error'))
     }
   }
+
+  private async getDockerInfo() {
+    if (!this.serverInfo?.homebridgeRunningInDocker) {
+      this.dockerStatusDone = true
+    } else {
+      try {
+        this.dockerInfo = await firstValueFrom(this.io.request('docker-version-check'))
+        this.dockerStatusDone = true
+      } catch (error) {
+        console.error(error)
+        this.$toastr.error(error.message, this.$translate.instant('toast.title_error'))
+      }
+    }
+  }
+
+  public toggleDockerExpand() {
+    this.dockerExpanded = !this.dockerExpanded
+  }
+
+  public dockerUpdateModal() {
+    const ref = this.$modal.open(InformationComponent, {
+      size: 'lg',
+      backdrop: 'static',
+    })
+
+    ref.componentInstance.title = this.$translate.instant('status.widget.info.docker_update_title')
+    ref.componentInstance.message = this.$translate.instant('status.widget.info.docker_update_message')
+    ref.componentInstance.markdownMessage2 = this.dockerInfo.latestReleaseBody
+    ref.componentInstance.subtitle = (this.dockerInfo.currentVersion && this.dockerInfo.latestVersion)
+      ? `${this.dockerInfo.currentVersion} → ${this.dockerInfo.latestVersion}`
+      : 'Unknown'
```

Comment:
maybe the unknown could go into a language string so it can be translated (or maybe it already exists?)